scripts/release.py

```diff
@@ -415,8 +445,32 @@ def build_parser():
         default='ssh://git@github.com/rackerlabs/docs-rpc.git',
     )
     parser.add_argument(
+        '--branch', help='Branch to update'
+    )
+    parser.add_argument(
         '--tag', type=Tag, required=True, help='Name of new tag.'
```

Comment:
Fair point.
@alextricity25 would you prefer remove this --tag argument, and read the tag from the file in the branch, and/or validate the equality of in-tree + cli provided tag?